content/cloud/develop.md

```diff
@@ -67,6 +73,8 @@ HTTP path: /...
 
 The command created all the files we need to build and run our Spin Application. Here’s the `spin.toml` file — the manifest file for a Spin application:
 
+<!-- @nocpy -->
```

Comment:
curious question, why `@nocpy` for this?

Reply:
Good catch! fixed it.